UI Test Runner/UITests/FormViewTests.swift

```diff
@@ -306,12 +306,12 @@ final class FormViewTests: XCTestCase {
         
         XCTAssertEqual(
             footer.label,
-            "Required"
+            "Date Entry is Required"
         )
         
         XCTAssertTrue(
-            calendarImage.isHittable,
-            "The calendar image isn't hittable."
+            calendarImage.exists,
+            "The calendar image doesn't exist."
```

Comment:
Hittable was working okay on Friday but seems to not be now. Checking for existence should be more stable.